### PR TITLE
Add support for new concordance model

### PR DESCRIPTION
**Pull request by nwrigh**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      CIRCLE_TEST_REPORTS: /tmp/test-reports
+      CIRCLE_ARTIFACTS: /tmp/artifacts
+      NEO4J_TEST_URL: "http://localhost:7474/db/data/"
+
+    steps:
+      - checkout
+      - run: |
+          go get -u github.com/kardianos/govendor
+      - run: |
+          go get -t -d -v ./...
```
Comment (david-smith-ft) [inline .circleci/config.yml]: Should be `govendor sync`

File: things/cypher.go [hunk 2]
```diff
@@ -30,56 +30,76 @@ func (cd cypherDriver) checkConnectivity() error { //TODO - use the neo4j connec
 	return neoutils.Check(cd.conn)
 }
 
-type neoReadStruct struct {
+type neoThing struct {
```
Comment (david-smith-ft) [inline things/cypher.go]: It'd be nice if this was formatted :)

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          goveralls -coverprofile=$CIRCLE_ARTIFACTS/coverage.out -service=circle-ci -repotoken=$COVERALLS_TOKEN
+
+      - store_test_results:
+          path: /tmp/test-reports
+
+      - store_artifacts:
+          path: /tmp/artifacts
+          destination: build
+  docker_build:
+    working_directory: /public-concordances-api
```
Comment (scott-ace-newton) [inline .circleci/config.yml]: things

File: things/handlers.go [hunk 4]
```diff
@@ -83,6 +84,8 @@ func GetThings(w http.ResponseWriter, r *http.Request) {
 	}
 
 	//if the request was not made for the canonical, but an alternate uuid: redirect
+
+	log.Printf("Url: %v Param: %v", uuid, thng.ID)
```
Comment (scott-ace-newton) [inline things/handlers.go]: Debug or going to print for every request?

File: vendor/vendor.json [hunk 5]
```diff
@@ -123,16 +141,16 @@
 			"revisionTime": "2017-03-28T16:39:54Z"
 		},
 		{
-			"checksumSHA1": "TC6pkBrqs0MYZEbikQcrloZw4Ec=",
+			"checksumSHA1": "diw1KueSVzsMHPCNDvj+xQiXSTE=",
 			"path": "github.com/Financial-Times/up-rw-app-api-go/rwapi",
-			"revision": "c7976a0b575e3cad95ec10ebca53798567b39845",
-			"revisionTime": "2016-08-24T12:50:00Z"
+			"revision": "d9d93a1f689538313d12fee6f5f10715cfe280e0",
+			"revisionTime": "2017-07-10T12:58:28Z"
```
Comment (scott-ace-newton) [inline vendor/vendor.json]: govendor fetch github.com/Financial-Times/up-rw-app-api-go/rwapi@1.0.0

File: vendor/vendor.json [hunk 6]
```diff
@@ -15,34 +15,52 @@
...
-			"checksumSHA1": "fg1dZ/MIpnMKoW0vJE2y5HpHXXc=",
+			"checksumSHA1": "5b1Y79xoNFuE/XXi5LDN3ys4OYA=",
 			"path": "github.com/Financial-Times/base-ft-rw-app-go/baseftrwapp",
-			"revision": "01975d9da464dc2b864f1a5cf603827fdaa8e661",
-			"revisionTime": "2016-09-09T09:52:18Z"
+			"revision": "27f4c78ff5bff819fea09533b4444693bca2136b",
+			"revisionTime": "2017-07-13T11:10:17Z"
+		},
+		{
+			"path": "github.com/Financial-Times/base-rw-neo4j",
```
Comment (scott-ace-newton) [inline vendor/vendor.json]: remove this. govendor remove github.com/Financial-Times/base-rw-neo4j

File: vendor/vendor.json [hunk 7]
```diff
@@ -15,34 +15,52 @@
...
 		},
 		{
-			"checksumSHA1": "qKDYaqwE0+Ulc1tiICBINAeutkg=",
-			"path": "github.com/Financial-Times/annotations-rw-neo4j/annotations",
-			"revision": "5e93af968065d5879b3c1846893dd36e5f8a5768",
-			"revisionTime": "2017-04-26T07:48:24Z",
-			"version": "=0.2.0",
-			"versionExact": "0.2.0"
+			"checksumSHA1": "59Q/nQ0hy76Vw6Ly2lPfm5ptQQs=",
+			"path": "github.com/Financial-Times/base-ft-rw-app-go",
```
Comment (scott-ace-newton) [inline vendor/vendor.json]: govendor fetch github.com/Financial-Times/base-ft-rw-app-go@1.0.1

**Pull request merged**